packages/webcomponents/src/elements.ts

```diff
@@ -206,6 +268,18 @@ if (Builder.isBrowser && !customElements.get(componentName)) {
         return;
       }
 
+      if (this.updateOnRouteChange) {
+        addHistoryChangeEvent();
+        window.addEventListener('popstate', this.updateFromRouteChange);
+        window.addEventListener('pushState', this.updateFromRouteChange);
+        window.addEventListener('replaceStateState', this.updateFromRouteChange);
+        this.subscriptions.push(() => {
+          window.removeEventListener('popstate', this.updateFromRouteChange);
+          window.removeEventListener('pushState', this.updateFromRouteChange);
+          window.removeEventListener('replaceStateState', this.updateFromRouteChange);
```

Comment:
```suggestion
        window.addEventListener('replaceState', this.updateFromRouteChange);
        this.subscriptions.push(() => {
          window.removeEventListener('popstate', this.updateFromRouteChange);
          window.removeEventListener('pushState', this.updateFromRouteChange);
          window.removeEventListener('replaceState', this.updateFromRouteChange);
```

Reply:
👍 sent fix over here - thanks for catching @mandx ! https://github.com/BuilderIO/builder/pull/268